gqlalchemy/memgraph.py

```diff
@@ -543,3 +556,146 @@ def create_relationship(self, relationship: Relationship) -> Optional[Relationsh
         )
 
         return self.get_variable_assume_one(results, "relationship")
+
+    def get_procedures(self, starts_with: str = None, update: bool = False) -> List["QueryModule"]:
```

Comment:
I now remember that you asked about `Optional` the other day. I read a bit about it and I learned that in Python>=3.9 it's the same whether you use `starts_with: str = None` or `starts_with: Optional[str]`. Do you agree or did I get it wrong?